src/RapidezStatamicServiceProvider.php

```diff
@@ -70,6 +76,7 @@ public function bootCommands() : self
     public function bootConfig() : self
     {
         $this->mergeConfigFrom(__DIR__.'/../config/rapidez/statamic.php', 'rapidez.statamic');
+        $this->mergeConfigFrom(__DIR__ . '/../config/rapidez/statamic/nav.php', 'rapidez.statamic.nav');
```

Comment:
Don't forget to update your bootPublishables, since 2 new config files have been added https://github.com/rapidez/core/pull/356/files#diff-251f27d5c5089c981407ccb64c4235715c0e2021ac4765b7f1bfea2e7795e608R113